scripts/run_radar_tests

```diff
@@ -1,7 +1,7 @@
 #! /bin/bash
 
 # clone a radar repo
-git clone https://github.com/zendesk/radar.git
+git clone -b pobrien/radar_test_changes_for_message_protocol_v2 git@github.com:zendesk/radar.git
```

Comment:
Yep.  I overlooked the fact that events **emitted** by the client that are listened to by other modules need to remain the same for now.  Reverted to master, made changes, tested, and the radar tests are now green.